app/models/concerns/oregon_digital/work_behavior.rb

```diff
@@ -6,8 +6,11 @@ module WorkBehavior
     extend ActiveSupport::Concern
     include OregonDigital::AccessControls::Visibility
 
+    attr_writer :graph_fetch_failures
```

Comment:
Changes to the work behavior will have no affect if `graph_fetch_failures` doesn't populate. Since you're calling `FetchFailedGraphWorker.perform_async` later, all changes in this file should be safe to drop